glotaran/analysis/result.py

```diff
@@ -1,12 +1,11 @@
 """The result class for global analysis."""
 
-import typing
 import os
+import typing
 
+import glotaran  # noqa F01
```

Comment:
this import order doesn't make sense. I would expect numpy and xarray to be imported first, as glotaran depends on them, also what does noqa F01 do here?

Reply:
Fixed in f155c7910add2fbe01b3b06c2e2fce2634ebc6f3